donate/core/models.py

```diff
@@ -118,6 +118,14 @@ def default_initial_amount(self, initial_currency_info, initial_frequency):
         """
         return sorted(initial_currency_info['presets'][initial_frequency])[1]
 
+    def default_other_amount(self, initial_currency_info):
```

Comment:
If the user shares the link with ?amount=other, we set the returned value to the minimum value of whichever currency we are using.

Then in donate_form.html, we are checking if the 'initial_amount', is == to the minimum amount for the currency. If so, highlight the "Other amount" field.

One thing I would like to note is that if the user does not type a custom amount and clicks "submit" the form still preforms as expected. It refreshes the page as the user left it blank.